Makefile

```diff
@@ -20,19 +21,40 @@ define docker_run
 		-v ${MAKEFILE}:${HOME}/Makefile \
 		-e EDITOR=vim \
 		-e POKY=${HOME}/poky \
-		-e TEMPLATECONF=${HOME}/poky/meta-gateway-ww/conf \
+		-e TEMPLATECONF=${HOME}/poky/meta-pelion-edge/conf \
 		--rm \
 		${USER}/ww-build-env:latest \
 		$(1)
 endef
 
 .PHONY: all
 all: Makefile .docker-image conf
-	cp ./mbed_cloud_dev_credentials.c ${POKY}/meta-gateway-ww/recipes-wigwag/mbed-edge-core/files/
+	if [ -e ./mbed_cloud_dev_credentials.c ]; then \
+		cp ./mbed_cloud_dev_credentials.c  ${POKY}/meta-pelion-edge/recipes-wigwag/mbed-edge-core/files/; \
+	fi
+	if [ -e ./update_default_resources.c ]; then \
+		cp ./update_default_resources.c ${POKY}/meta-pelion-edge/recipes-wigwag/mbed-edge-core/files/; \
+	fi
 	$(call docker_run, make bb/${IMAGE_RECIPE})
 
+flash-%: all
+	$(foreach dev, $(wildcard /dev/${*}*),\
+		sudo umount ${dev} || true; \
+	)
+	if which bmaptool ; then\
```

Comment:
On the one hand, this bothers me because it assumes that a user will always want to flash a drive with bmaptool if it's installed, but on the other hand, it does add a makefile command that was probably needed, with an assumption that isn't unreasonable.

I'm fine asis without changes w.r.t this addition, just conflicted.